xc/xc7/tests/dram/CMakeLists.txt

```diff
@@ -28,14 +28,14 @@ function(dram_tests dram_num_instances dram_mode outpad inpad enable_vivado_targ
 endfunction()
 
 # dram_tests(<dram_num_instances> <dram_mode> <syn_outpad_assert_usage> <syn_inpad_assert_usage> <enable_vivado_target>)
```

Comment:
Are there any `dram_tests` which set `<enable_vivado_target>` to `N`?

Reply:
There are no such tests. This option was introduced in https://github.com/SymbiFlow/symbiflow-arch-defs/pull/1268 where some tests were actually disabled. This was then fixed in https://github.com/SymbiFlow/symbiflow-arch-defs/pull/1356 but the option was not deleted. Not sure if this is required at this point. I added a commit that deletes this option in case it is no longer needed.